commands/test/test-command.ts

```diff
@@ -93,6 +93,77 @@ const job = new cron.CronJob('0 0 * * * *', checkForFeedbackRequests); // checks
 job.start();
 
 
+export const sendTestButton = async () => {
+    const button = new ButtonBuilder()
+        .setCustomId('start_test')
+        .setLabel('Start Test')
+        .setStyle(ButtonStyle.Danger);
+
+    const actionRow = new ActionRowBuilder<ButtonBuilder>()
+        .addComponents(button);
+
+    const guildId = process.env.GUILD_ID;
+    if (!guildId) throw new Error('GUILD_ID is not defined in .env');
+
+    const guild: Guild | undefined = client.guilds.cache.get(guildId);
+    if (!guild) throw new Error('Guild not found');
+
+    (guild.channels.cache.get("1135557183845711983") as TextChannel).send({ components: [actionRow] });
```

Comment:
Is this the correct Id for the main server?

Reply:
Or what channnel Id is this?

Reply:
Maybe add a comment for what channel it is to make it more readable

Reply:
yes its the correct id, added comment